src/components/markdown-input.js

```diff
@@ -79,7 +79,7 @@ function MarkdownInput({
     return (
       <div className="MarkdownInput-inlineInputContainer" ref={inputContainerRef}>
         {!debounced && <Input {...inputProps} />}
-        {!!debounced && <DebouncedInput apiRef={debouncedInputApiRef} {...inputProps} elementType={Input} />}
+        {!!debounced && <DebouncedInput apiRef={debouncedInputApiRef} {...inputProps} />}
```

Comment:
**Actually** it seems we would now be left with just one case in which `elementType` is passed to `DebouncedInput`, and that type is `TextArea`.
Since `NeverScrollingTextArea` does support the `debounced` mode, we could replace maybe that one usage and drop `elementType` support altogether? This would also be in line with the `MarkdownInput` change you did, with using its `debounce` support.
Think that's reasonable to do?

Reply:
"Think that's reasonable to do?", she asked. "No, it doesn't", she noticed, as the component using the `DebouncedInput` with `elementType` set to `TextArea` is the `NeverScrollingTextArea` itself 🤦‍♀️